### PR TITLE
Clarify how you can get help

### PR DESCRIPTION
**Pull request by aub**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ Each directory contains a homework problem, four in total. Please complete one--
 more--of the four before submitting. Make sure to read the README in each directory,
 it will give you important information on completing each problem.
 
-We're on the honor system here. Don't get help from, or give help to, another
-person or a resource on the internet. Do the work yourself. It's okay if you
-need to consult language docs or the MDN or similar.
+We're on the honor system here, and the goal is to show what you're capable of.
+Feel free to use any resources you like in solving the problems
+but don't ask others for help or make use of solutions you may find out there on the net.
```
Comment (earthboundkid) [inline README.md]: How about "pre-existing solutions"? Finding a post about how to add an event listener for a remove button is okay, but finding someone else's homework is not.

**Comment by aub** [comment]
Sure, @carlmjohnson how about now?

**Comment by earthboundkid** [comment]
Is this too wordy?

> We're on the honor system here, and the goal is to show what you're capable of. Do the work yourself, just as you would as an employee. It's okay if you need to consult language docs, the MDN, or similar resources, but don't ask others for help or make use of pre-existing solutions you may find out there on the net.

**Comment by aub** [comment]
@carlmjohnson I'm just not sure how it's an improvement... it may make people think that the only things they can use are language docs or the MDN when in fact I think we're fine with them using just about anything other than somebody else's solution they dug up.

**Comment by earthboundkid** [comment]
> We're on the honor system here, and the goal is to show what you're capable of. Do the work yourself, just as you would as an employee. Feel free to use any resources you like in solving the problems, but don't ask others for help or make use of pre-existing solutions to these problems you may find out there on the net.

I'm adding "to these problems" to signal it's okay to use a StackOverflow post as long as it's about another problem, not "help me do Ad Hoc's Homework".

**Comment by aub** [comment]
@carlmjohnson ok, I stripped it down and tried again. I want it to be short and totally clear.

**Comment by earthboundkid** [comment]
Boom. Perfect.
